google-http-client/src/main/java/com/google/api/client/http/HttpResponseException.java

```diff
@@ -123,8 +131,22 @@ public HttpHeaders getHeaders() {
    *
    * @since 1.14
    */
-  public final String getContent() {
-    return content;
+  public final String getContent() throws IOException {
+    if (content == null) {
+      return null;
+    }
+    ByteArrayOutputStream out = new ByteArrayOutputStream();
+    ByteStreams.copy(new ByteArrayInputStream(content), out);
+    return out.toString(charset.name());
+  }
+
+  /**
+   * Returns the HTTP response or {@code null} for none as an input stream.
+   *
+   * @since 1.28
+   */
+  public final InputStream getContentAsInputStream() {
```

Comment:
I think that we ought to be more direct here and just do this:

```
public final byte[] getRawContent() {
   return content;
}
```

Reply:
This is a good point